### DISCUSSION TITLE
win12今年就出了

### DISCUSSION THREAD

**Discussion by ghost** [win12-online/win12#386] [Ideas]
So

**Comment by User782Tec** [comment]
改名win13，继续开发

**Reply by kjmjh** [reply]
6

**Comment by NebulaTechs** [comment]
没出，改为了所谓的Windows11 24H2

**Comment by lingbopro** [comment]
> 改名win13，继续开发

那如果巨硬发了Windows13呢（  
不如叫win114514罢，巨硬肯定更不到这么大的版本号的（悲

**Comment by Mbilse** [comment]
> > 改名win13，继续开发
> 
> 那如果巨硬发了Windows13呢（  
> 不如叫win114514罢，巨硬肯定更不到这么大的版本号的（悲

《Windows 114514 家庭版 1809》

**Comment by Mbilse** [comment]
> > > 改名win13，继续开发
> > 
> > 那如果巨硬发了Windows13呢（  
> > 不如叫win114514罢，巨硬肯定更不到这么大的版本号的（悲
> 
> 《Windows 114514 家庭版 1809》

还是像win93，96那样叫一个老巨硬想不到的名字吧：win9？
